components/brave_new_tab_ui/components/default/rewards/notification.tsx

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this file,
+* You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+import {
+  Title,
+  SubTitle,
+  CloseIcon,
+  Content,
+  NotificationWrapper,
+  NotificationButton
+} from './style'
+import { CloseStrokeIcon } from 'brave-ui/components/icons'
+import { getLocale } from '../../../../common/locale'
+
+interface NotificationProps {
+  grant: NewTab.GrantRecord
+  onDismissNotification: (id: string) => void
+}
+
+export default class RewardsNotification extends React.PureComponent<NotificationProps, {}> {
+
+  dismissNotification = () => {
+    this.props.onDismissNotification(this.props.grant.promotionId)
+  }
+
+  onNotificationAction = () => {
+    this.dismissNotification()
+    window.open(`chrome://rewards#grant_${this.props.grant.promotionId}`, '_blank')
```

Comment:
the popup API is now merged if you want to use that

Reply:
now using the popup api and doing this via the panel